vendorweapons.php

```diff
@@ -8,35 +9,75 @@
+        <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+        <link rel="stylesheet" type="text/css" href="./static/css/main.css">
+        <link rel="stylesheet" type="text/css" href="./static/css/nav.css?ver=3">
+        <link rel="stylesheet" type="text/css" href="./static/css/tables.css?ver=5">
+        <link rel="stylesheet" type="text/css" href="./static/css/jquery.dataTables.css">
+        <!-- JS -->
+        <script src="./static/js/jquery-3.2.1.min.js"></script>
+        <script src="./static/js/bootstrap.min.js"></script>
+        <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.15/js/jquery.dataTables.js"></script>
+    </head>
+
+    <body>
+        <div class="masthead-weapons"> </div>
+        <div id="nav-wrap">
+            <div id="main-nav" class="navbar">
+                <div class="navbar-inner">
+                    <div class="container">
+                        <div class="navbar-header">
+                            <div class="navbar-brand navbar-brand-centered"></div>
+                        </div>
+                    </div>
+                </div>
+                <!-- navbar -->
+            </div>
+        </div>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-md-2 sidebar1">
+                    <div class="container sidebar1" id="sidebar">
+                        <div class="logo"> <img src="./static/img/Raid_Emblem.png" class="rounded" alt="Logo" style="max-width:175px; border:0; padding-left: 40px;"> </div>
+                        <br>
+                        <br>
+                        <br>
+                        <br>
+                        <div class="left-navigation">
+                            <ul class="list">
+                                <h5><strong>LINKS</strong></h5>
+                                <li><a href="https://goo.gl/xgpzbX">Discuss this on Reddit</a></li>
+                                <li><a href="vendorweapons.php">Vendor Weapons</a></li>
+                            </ul>
+                            <br>
+                            <ul class="list">
+                                <h5><strong>/r/DTG LINKS</strong></h5>
+                                <li><a href="https://www.reddit.com/r/DestinyTheGame">Back to the subreddit</a></li>
+                                <li><a href="https://discord.gg/DestinyReddit">Destiny Discord</a></li>
+                                <li><a href="https://www.reddit.com/message/compose?to=%2Fr%2FDestinyTheGame&subject=DestinyReddit%20Site">Contact Info</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-10 main-content">
+                    <div class="container-fluid" id="wrapper">
+                        <h1 class="title">Vendor Weapons - 6/6/2017</h1>
```

Comment:
Before we go beta will have to have it auto insert the Tuesday Week date automatically. Doesn't need to be changed today.